Examples/Examples/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This file looks redundant with https://github.com/mapbox/mapbox-maps-ios/pull/193/files#diff-02029dab4ca42e1076eb249084403aac9d416ba002e7bcb053b856d94c51bd26R1

Reply:
Removed.